helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -653,6 +653,13 @@ private RollingUpdateOp rollingUpdateTimedoutError(final RollingUpdateOpFactory
               RollingUpdateError.IMAGE_PULL_FAILED,
               metadata);
     }
+    if (!taskStatus.getContainerError().equals("")) {
```

Comment:
In `TaskStatus.java` containerError is created like:
```java
this.containerError = Optional.fromNullable(builder.containerError).or("")
```

Is it safe for me to assume that means `taskStatus.getContainerError()` should never return null? If so, is there still value in using `isNullOrEmpty()`? I notice there's also an `isEmpty()` method available on `taskStatus.getContainer()` already.